lms/djangoapps/certificates/apis/v0/tests/test_views.py

```diff
@@ -65,29 +106,115 @@ def test_permissions(self):
         self.assertEqual(resp.status_code, status.HTTP_401_UNAUTHORIZED)
 
         # another student
-        self.client.login(username=self.student_no_cert.username, password='test')
+        self.client.login(username=self.student_no_cert.username, password=USER_PASSWORD)
         resp = self.client.get(self.get_url(self.student.username))
         # gets 404 instead of 403 for security reasons
         self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
         self.assertEqual(resp.data, {u'detail': u'Not found.'})  # pylint: disable=no-member
         self.client.logout()
 
         # same student of the certificate
-        self.client.login(username=self.student.username, password='test')
+        self.client.login(username=self.student.username, password=USER_PASSWORD)
         resp = self.client.get(self.get_url(self.student.username))
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
         self.client.logout()
 
         # staff user
-        self.client.login(username=self.staff_user.username, password='test')
+        self.client.login(username=self.staff_user.username, password=USER_PASSWORD)
         resp = self.client.get(self.get_url(self.student.username))
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
 
+    def test_inactive_user_access(self):
+        """
+        Verify inactive users - those who have not verified their email addresses -
+        are allowed to access the endpoint.
+        """
+        self.client.login(username=self.student.username, password=USER_PASSWORD)
+
+        self.student.is_active = False
+        self.student.save()
+
+        resp = self.client.get(self.get_url(self.student.username))
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+
+    def test_dot_valid_accesstoken(self):
+        """
+        Verify access with a valid Django Oauth Toolkit access token.
+        """
+        # make sure the client is not logged in
+        self.client.logout()
```

Comment:
You can wrap this into a test helper method that can be used to de-dup these changes:

```python
def assert_oauth_status(self, expected_status):
  self.client.logout()
  auth_header = "Bearer {0}".format(self.dot_access_token)
  response = self.client.get(self.get_url(self.student.username), HTTP_AUTHORIZATION=auth_header)
  self.assertEqual(response.status_code, expected_status)
```